### PR TITLE
remove second dockerfile

### PR DESCRIPTION
**Pull request by anwilkie**
# What and why?
Removing the dockerfile in the root directory to ensure consistency with other repos.

# How to test?
Check GA build

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-313)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,13 +7,13 @@ start:
 	pipenv run python run.py
 
 lint:
-	pipenv check -i 51668
+	pipenv check -i 70612 -i 71064 -i 70624
```
Comment (arroyoAle) [inline Makefile:10]: again there is an update for requests (71064) can be a separate card

**Review by arroyoAle — APPROVED** [review]
Works fine just a little comment

File: Pipfile [hunk 2]
```diff
@@ -24,6 +23,8 @@ numpy = "*"
 google-cloud-pubsub = "*"
 flask-sqlalchemy = "*"
 flask = "*"
+urllib3 = "*"
```
Comment (matthew-robinson-ons) [inline Pipfile:26]: we shouldn't be importing urllib3 it is managed as a subdependency of a number of packages, e.g. requests

**Review by matthew-robinson-ons — CHANGES_REQUESTED** [review]
Just the one comment around urlib, the vulnerability is now fixed on main so after merging the import can be removed

**Pull request closed without merging**